compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmTerminatorGen.java

```diff
@@ -649,6 +659,46 @@ private void genJICallTerm(JIMethodCall callIns, int localVarOffset) {
         this.mv.visitLabel(notBlockedOnExternLabel);
     }
 
+    private void loadFuncPathParams(MethodVisitor mv, BInvokableTypeSymbol invokableSymbol) {
+        List<BVarSymbol> params = new ArrayList<>();
+        if (invokableSymbol != null) {
+            for (BVarSymbol param : invokableSymbol.params) {
+                SymbolKind paramKind = param.getKind();
+                if (paramKind != SymbolKind.PATH_PARAMETER && paramKind != SymbolKind.PATH_REST_PARAMETER) {
+                    break;
+                }
+                params.add(param);
+            }
+        }
+
+        mv.visitLdcInsn((long) params.size());
+        mv.visitInsn(L2I);
+        mv.visitTypeInsn(ANEWARRAY, FUNCTION_PARAMETER);
+        for (int i = 0; i < params.size(); i++) {
```

Comment:
We already have loadFunctionParameters in jvmtypegen. can we reuse those logic?

Reply:
Yes, I will refactor to remove duplicate parts